src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -236,6 +236,23 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
         )}
         text="Test Unlock Token"
       />
+      <ActionButton
+        submit={ActionTypes.ACTION_MOVE_FUNDS}
+        error={ActionTypes.ACTION_MOVE_FUNDS_ERROR}
+        success={ActionTypes.ACTION_MOVE_FUNDS_SUCCESS}
+        transform={pipe(
+          mergePayload({
+            colonyAddress: colony.colonyAddress,
+            colonyName: colony.name,
+            fromDomain: colony.domains?.items.find((d) => d?.isRoot),
+            toDomain: colony.domains?.items.find((d) => !d?.isRoot),
```

Comment:
For testing, this will attempt to transfer from root to the first non-root domain it can find